LICENSE.md

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright © `2019` `Arron Hunt <arronjhunt@gmail.com>`
+Copyright © `2021` `Taskade Inc <amos@taskade.com>`
```

Comment:
Let's keep the original author credit and don't overwrite it. We can addon if we want, but not replace the original author.